homepage/views.py

```diff
@@ -136,6 +136,21 @@ def sign_out(request):
     return redirect('landing')
 
 
+@login_required(login_url='/users/sign_in/')
+def follow_course_action(request, course_id):
+    course = Course.objects.get(course_id=course_id)
```

Comment:
```suggestion
try:
    course = Course.objects.get(course_id=course_id)
except ObjectDoesNotExist:
        return redirect('/courses/')
```
In case someone typing an invalid URL